thunder/core/jit_ext.py

```diff
@@ -1347,6 +1348,26 @@ def bind_inputs(name, trace, input_vars, input_proxies):
     trace.args = input_proxies
 
 
+def _get_process_group_from(*fn_and_args) -> Optional["ProcessGroup"]:
+    # `ddp` and `fsdp` transforms add attribute `procses_group_for_ddp`
+    # on the Module that they wrap. This module could be passed to `thunder.jit`
+    # as the function to be jitted or as an argument of the function to be jitted.
+    pgs = []
+    for fn_or_arg in fn_and_args:
+        pg = getattr(fn_or_arg, "process_group_for_ddp", None)
+        if pg is not None:
+            pgs.append(pg)
+
+    # attribute wasn't found, return None.
+    if pg == []:
+        return None
+
+    # check all pgs are equal
+    if not reduce(operator.eq, pgs):
```

Comment:
Does this work though? operator.eq is mapping to bool.

Reply:
This doesn't work, thanks for catching this and help offline!